gqlalchemy/query_builder.py

```diff
@@ -139,15 +147,68 @@ def construct_query(self) -> str:
 
 
 class WhereConditionPartialQuery(PartialQuery):
-    def __init__(self, keyword: str, query: str):
-        super().__init__(DeclarativeBaseTypes.WHERE)
+    _LITERAL = "literal"
+    _EXPRESSION = "expression"
+    _LABEL_FILTER = ":"
 
-        self.keyword = keyword
-        self.query = query
+    def __init__(self, item: str, operator: str, keyword: Where = Where.WHERE, is_negated: bool = False, **kwargs):
+        super().__init__(type=keyword.name if not is_negated else f"{keyword.name} {Where.NOT.name}")
+        self.query = self._build_where_query(item=item, operator=operator, **kwargs)
 
     def construct_query(self) -> str:
         """Constructs a where partial query."""
-        return f" {self.keyword} {self.query} "
+
```

Comment:
Delete this line. Or we discuss whether we keep the empty line after docstring.

Reply:
There are a couple of occurrences of this, so let's discuss it. I'll put this in the channel.